src/antsibull_build/changelog.py

```diff
@@ -62,6 +95,10 @@ class ChangelogData:
     generator: ChangelogGenerator
     generator_flatmap: bool
 
+    remove_collection_changelog_entries: (
+        dict[str, dict[SemVer, RemoveCollectionVersionSchema]] | None
+    ) = None
```

Comment:
```suggestion
    remove_collection_changelog_entries: (
        dict[str, dict[SemVer, RemoveCollectionVersionSchema]] | None
    )
```

This doesn't need to be a class variable and is already set to `None` in `__init__()`